_src/css/_section.scss

```diff
@@ -0,0 +1,39 @@
+.section,.section--reverse{
+  display: flex;
+  justify-content: space-around;
+  height: 85vh;
+  background-color: lightblue;
+  
+  &__side{
+    display:flex;
+    align-items: center;
+  }
+
+  &__image{
+      color: white;
+      display:flex;
+      justify-content: center;
+      align-items: center;
+      width: 200px;
+      height: 200px;
+      border: 3px solid white;
+      border-radius: 100%;
+  }
+    
+  &__text{
+    display:flex;
+    align-items: center;
+    margin: 0;
+  }
+
+  &__paragraph{
```

Comment:
You can replace the display for 'width' so we will be able to manipulate better the text inside the relative parent